cognite/client/data_classes/simulators/runs.py

```diff
@@ -424,10 +440,56 @@ def dump(self, camel_case: bool = True) -> dict[str, Any]:
         output["outputs"] = [output_.dump(camel_case=camel_case) for output_ in self.outputs]
         return output
 
+    def to_pandas(  # type: ignore [override]
+        self,
+    ) -> pandas.DataFrame:
+        """Convert the simulation run data to a pandas DataFrame.
+
+        Returns:
+            pandas.DataFrame: The dataframe.
+        """
+        pd = local_import("pandas")
+
+        def _create_row(item: SimulationInput | SimulationOutput) -> dict:
+            """Create a row dictionary for an input or output item."""
+            item_type = (isinstance(item, SimulationInput) and "Input") or "Output"
```

Comment:
Any reason not to use a dump method? What happens if more properties are added to the simulation input?